packages/server/assets/gatekeeperCore/typedefs/workspaceSeats.graphql

```diff
@@ -4,7 +4,21 @@ enum WorkspaceSeatType {
 }
 
 extend type WorkspaceCollaborator {
-  seatType: WorkspaceSeatType!
+  seatType: WorkspaceSeatType
```

Comment:
made it nullable in case module is disabled

Reply:
we should pro move things around. If you are on a new plan workspace, you will 100% have a seat type, no matter if the gatekeeper module is enabled or not.

But for now, if you are on an old plan, we do not assign seats, so the typing change is technically correct.